ietf-optical-impairment-topology.yang

```diff
@@ -1035,74 +1025,144 @@ Concentratedloss)";
    }
   }
 
-  augment "/nw:networks/nw:network/nt:link/tet:te"
-   + "/tet:te-link-attributes"   {
-   when "/nw:networks/nw:network/nw:network-types"
-    +"/tet:te-topology/"
-    +"optical-imp-topo:optical-impairment-topology" {
+  augment "/nw:networks/nw:network/nw:node" {
+    when "../nw:network-types/tet:te-topology" +
+         "/optical-imp-topo:optical-impairment-topology" {
+      description
+        "This augment is only valid for Optical Impairment.";
+    }
     description
-      "This augment is only valid for Optical Impairment.";
-   }
-   description "Optical Link augmentation for impairment data.";
-   container OMS-attributes {
-       config false;
-       description "OMS attributes";
-     uses oms-general-optical-params;
-     uses media-channel-groups;
-     uses oms-element;
-     }
+      "Node augmentation for optical impairments data.";
+    list transponder {
+      key "transponder-id";
+      config false;
+      description "list of transponder";
+      leaf transponder-id {
+        type uint32;
+        description "transponder identifier";
+      }
+
+      list transceiver {
+        key "transceiver-id";
+        config false;
+        description "list of transceiver related to a transponder";
+        leaf transceiver-id {
+          type uint32;
+          description "transceiver identifier";
+        }
+        leaf supported-termination-type {
```

Comment:
this is about the transceiver capability. If a '3r-or-tunnel' capable trasceiver is used as transceiver, will this value change ? (I supposed not, because it is config False)

Reply:
IMHO, this should represent the capability of the transceiver independently from its actual use: if it is used as a termination, it is still capable to be re-configured as a 3R. It seems worthwhile clarifying this in the description.

Reply:
Shall we add, as made in case of "mode" for transceiver , add a configured-termination-capability?
For mode we have a configured-mode pointer to transceiver and related value for mode

Reply:
I think this is a good idea, I prefer that this is explicit rather than implicit from a given tunnel configuration external from this model. eg:
termination-type- capability (instead of supported-termination-type)
(config false)
configured-termination-type
(config false)

Reply:
we need to consider also the need for an attribute providing the "association" with the other transceiver in the 3R configuration.
Need to check if both the attributes are really needed to avoid inconsistency.

Reply:
Addressed in https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-optical-impairment-topology-yang/pull/78/commits/942095c8f25e0ecb5303a4ae26ef49d5ae936162

More discussion is needed to understand how to associate the transceivers used in a 3R configuration (especially in case of multi-carrier signals): see issue #83